osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -58,6 +61,11 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
             BeatmapSetInfo beatmapSetInfo = beatmap.BeatmapSetInfo;
             BeatmapInfo beatmapInfo = beatmap.BeatmapInfo;
 
+            string bpm = getBPMRange(beatmap);
+            string length = TimeSpan.FromMilliseconds((beatmap.Beatmap.HitObjects.Last().EndTime - beatmap.Beatmap.HitObjects.First().StartTime)).ToString(@"m\:ss");
+            string hitCircles = "" + beatmap.Beatmap.HitObjects.Count(h => h.GetType().ToString().Equals("osu.Game.Modes.Osu.Objects.HitCircle"));

```

Comment:
Nuke these type name comparisons and use instead `beatmap.Beatmap.HitObjects.Count(o => o is HitCircle)`, etc.
Use ToString instead of prepending empty string while you're at it, too, because that's fugly.
The lines should read
```cs
string hitCircleCount = beatmap.Beatmap.HitObjects.Count(o => o is HitCircle).ToString();
```